README.md

```diff
@@ -5,10 +5,11 @@ It's that time of year again! Time to get rewarded for contributing to the Open
 This repo seeks to document all of the companies giving away swag for Hacktoberfest 2020 and how you can get your hands on some free gifts for your time and help.
 
 ## Hacktoberfest is about making meaningful contributions to open source projects. SPAM for the sake of a shirt will not be tolerated
-
+ 
```

Comment:
You have a space inserted here. 🙃 That's all! I'll remove this one blank space for you, but that is what was causing the Code Climate flag.